src/components/ServiceDetails/PricingDetails/index.js

```diff
@@ -15,7 +15,7 @@ const PricingDetails = ({ classes, price_model }) => {
         <Price unit="agi tokens" value="0.000001" />
       </div>
       <p>
-        <i className="fas fa-info-circle"></i>
+        <i className="fas fa-info-circle" />
```

Comment:
We could use `Icon` component here.

Reply:
[done](https://github.com/Vivek205/refactored-dapp/pull/26/commits/9f7da351e1fb76764f2e27e1d9bacc88f3812973)